### ISSUE TITLE
Naming conventions for event handlers

### ISSUE THREAD

**Issue by YijinL** [HubTurbo/addressbook#44]
Currently, all event handlers are named `handle<event name>`.

This is redundant because:

1. `@Subscribe` indicates that this method is an event handler, and
2. The parameter type indicates the event type being handled.

See [here](https://github.com/google/guava/wiki/EventBusExplained#why-use-an-annotation-to-mark-handler-methods-rather-than-requiring-the-listener-to-implement-an-interface) for recommended naming of`EventBus` subscribers.
Here is an example change we could do. [this handler here:](https://github.com/HubTurbo/addressbook/blob/c1ec46d5ae3d992a4a1dbb2a64e46252846d2612/src/main/java/address/sync/SyncManager.java#L67-L68)
```java
@Subscribe
public void handleLocalModelChangedEvent(LocalModelChangedEvent lmce) {
    requestExecutor.execute(new CloudUpdateTask(this.cloudSimulator, lmce.personData, lmce.groupData));
}
```
would be better named `syncLocalChangesToCloud`. This way we don't have to guess it's purpose from only it's class name and event name.

**Comment by damithc** [comment]
Good point. One concern is the method may have to do multiple things to respond to the event, and hence it might be difficult to find an alternative name. i.e. handleEventX may be the most suitable description for the method.  
Let's revisit this issue after we have more handlers.
